src/components/form/ColorPresetsPicker.svelte

```diff
@@ -0,0 +1,112 @@
+<svelte:options immutable={true} />
+
+<script lang="ts">
+  import { createEventDispatcher } from 'svelte';
+  import { getTarget } from '../../utilities/generic';
+  import { tooltip } from '../../utilities/tooltip';
+  import Menu from '../menus/Menu.svelte';
+  import ColorPicker from './ColorPicker.svelte';
+  export let value: string = '';
+  export let tooltipText: string = 'Color';
+  export let presetColors: string[] = ['#ef8b8c', '#febd85'];
```

Comment:
Please add some newlines in this file between the different types

Reply:
Oh weird I don't know how I lost all of these newlines, good catch